content/projects/katas/level-1/_index.md

```diff
@@ -204,15 +213,15 @@ time
 
 Write a function `combine` that combines two lists by taking alternate elements and returns the result. You can assume the lists are of equal size.
 
-Your combine function should be able to work with lists of unequal size.
+Your combine function should be able to work with lists/arrays of unequal size. 
```

Comment:
VERY IMPORTANT

I think I have mentioned this before on another project. Please please rememmber that this is true for ALL java projects:

We must never ever ask the learner for a Luist/asrray in JAva. We must tell them that we want an artray or we must tell them that we want a list. We must not tell them to pick the data type themselves.

Why? Because we want the automarker to work.